pkgs/data/icons/tela-icon-theme/default.nix

```diff
@@ -17,6 +17,10 @@ stdenv.mkDerivation rec {
 
   dontDropIconThemeCache = true;
 
+  # These fixup steps are slow and unnecessary.
```

Comment:
In particular: hydra workers tend to be overcontended on disk I/O anyway, so reducing it is helpful, including full-output-scans for packages with many many files, like this one.